core/controllers/acl_decorators.py

```diff
@@ -1508,7 +1514,7 @@ def test_can_edit(self, topic_id, **kwargs):
             raise base.UserFacingExceptions.PageNotFoundException
 
         if topic_services.check_can_edit_topic(self.user, topic_rights):
-            return handler(self, topic_id, **kwargs)
+            return handler(self, topic_id, *args, **kwargs)
```

Comment:
Just wondering, why add *args? (What issue does this fix?)

Reply:
When writing tests for `SuggestionToTopicActionHandler` I saw that the decorator `can_edit_topic` should accept two args -- target_id and suggestion_id. This is why I added *args. Without this I was getting an error -- put takes exactly 3 arguments (2 given) [or something similar]

Reply:
Ah ok, thanks! Good that you caught that.

@nithusha21 @aks681 could you please advise on whether this is expected? I am a bit surprised this wasn't caught before (e.g. while testing manually).

Reply:
Hmmm, this is expected. I seem to have done this change in can_edit_exploration but not in can_edit_topic. I am surprised how it worked before while we manually tested (as I clearly remember testing the complete flow for question creation using suggestions, which uses this handler). This has never been released, however, and users do not have access to it. 

The error @lilithxxx mentions is correct and should have been thrown. This flow also had tests written in controllers/suggestion_test.py. I'll check and get back with any extra information.

But anyway! Great catch, and thanks for the fix!